packages/popper/src/use-popper.ts

```diff
@@ -98,6 +99,7 @@ export type ArrowCSSVarProps = {
 
 export function usePopper(props: UsePopperProps = {}) {
   const {
+    isEnabled = true,
```

Comment:
What is the benefit in terms of performance when setting this by default to true?

A simple tooltip like this
```tsx 
<Tooltip label="Hey, I'm here!" aria-label="A tooltip">
  Hover me
</Tooltip>
```

will still create a popper instance, or?

Reply:
I am passing `isEnabled: isOpen` to `usePopper` in the `useTooltip` hook, so the `popper.js` instance will not be created until the tooltip is actually visible.

Reply:
Ah it is not the `isOpenProp` lol my bad - i was like wtf how should this work :D